src/frontend/charCounter.ts

```diff
@@ -0,0 +1,7 @@
+
+/*
+    Script to count the number of letters typed.
+*/
+function ShowLength( str : string ) {
+    document.getElementById("inputlength").innerHTML = str.length + "/250";
```

Comment:
```suggestion
    document.getElementById("inputlength").textContent = str.length + "/250";
```
Although this is not a real security concern as you're only inserting numbers here, but it's still best practice to use `textContent` instead of `innerHTML` if you're inserting something which should not be rendered as HTML.